client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -11,41 +11,42 @@ import GridiconMoney from 'gridicons/dist/money';
  */
 import edit from './edit';
 import save from './save';
-import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+/**
+ * Styles
+ */
+import './editor.scss';
 
 registerBlockType( 'jetpack/simple-payments', {
 	title: __( 'Payment button' ),
 
 	description: __(
-		'Simple Payments lets you create and embed credit and debit card payment buttons on your WordPress.com and Jetpack-enabled sites with minimal setup.'
+		'Simple Payments lets you create and embed credit and ' +
+			'debit card payment buttons on your WordPress.com and ' +
+			'Jetpack-enabled sites with minimal setup.'
 	),
 
 	icon: <GridiconMoney />,
 
 	category: 'jetpack',
 
-	keywords: [ __( 'simple payments' ), 'PayPal' ],
+	keywords: [ __( 'Simple Payments' ), _x( 'shop', 'block search term' ), 'PayPal' ],
 
 	attributes: {
 		currency: {
 			type: 'string',
-			default: 'USD',
```

Comment:
Why are we dropping this default?

I think defaulting to USD is good and matches Calypso:

https://github.com/Automattic/wp-calypso/blob/1061190e69831ca28f31b12644943de1b2f614ac/client/components/tinymce/plugins/simple-payments/dialog/index.jsx#L155

https://github.com/Automattic/wp-calypso/blob/1061190e69831ca28f31b12644943de1b2f614ac/client/components/tinymce/plugins/simple-payments/dialog/index.jsx#L222